canonical/canonical.com
# Build /solutions/ai + modal

_Needs pdm approval before merging_ 

## Done

- Added new page as per [design](https://www.figma.com/file/CzC9ROWlVygyk1pUuEHA7U/Canonical.com%2Fsolutions%2Fai?node-id=0%3A1&mode=dev) and [doc](https://docs.google.com/document/d/1AtAApqxX9cbWf4a3cakEYbk7OoTfdBBi7FlA3WZz5ZM/edit)
- Added new modal as per [doc](https://docs.google.com/document/d/1rlAO5gLtxsmi3la_yk2rCBOTXBAph2pkRhwVyu6Ua-o/edit)
- Drive by:
  - Moved remaining /solutions forms to their own respective folders for consistency     

## QA

- View the site locally in your web browser at: https://canonical-com-1222.demos.haus/solutions/ai
- Check against doc and design
  - Be sure to check small / medium screens 
- Check that the modal copy matches the doc and that everything submits as expected 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9435
